simtools/db/db_handler.py

```diff
@@ -1420,11 +1429,19 @@ def get_collections(self, db_name=None):
         -------
         list
             List of collection names
+        model_collections_only: bool
+            If True, only return model collections (i.e. exclude fs.files, fs.chunks)
```

Comment:
please include "metadata" in this example

Reply:
Done.